src/js/components/presentation/inputs/range-input.jsx

```diff
@@ -4,44 +4,48 @@ import PropTypes from 'prop-types'
 import 'styles/components/inputs'
 
 const RangeInput = (props) => {
-  let {
-    className, errors,
-    id, label, placeholder,
-    required, validateAs, value, ...rest } = props;
+    let {
+        className, errors, formErrors,
+        id, label, placeholder, showLabel,
+        required, validateAs, value, ...rest
+    } = props;
 
-  let attr = {};
+    let attr = {};
 
-  if (required) {
-    attr['required'] = true;
-    attr['aria-required'] = true;
-  }
+    if (required) {
+        attr['required'] = true;
+        attr['aria-required'] = true;
+    }
 
-  return (
-    <label className='input__label input__label--wrap'>
-      { label }{ attr.required && <span>{ '\u00A0' }*<span className="u-sr-only"> required field</span></span> }
-      <input type='range' value={ props.value } name={ id } id={ id } onChange={ props.onChange } min='0' max={ props.total }
-        className={ `input input--range ${ className ? className : '' } ${ formErrors && formErrors[id] ? 'input--invalid' : '' }` }
-        data-validate={ validateAs }  placeholder={ placeholder } value={ value } step='0.01' onMouseOut={ props.onBlur }
-        { ...attr } />
-      { formErrors && formErrors[id] ?
-        <div className='input__error'>{ errors }</div>
-      :
-        '' }
-    </label>
-  )
+    return (
+        <label className='input__label input__label--wrap'>
+            { label && showLabel }{ attr.required &&
```

Comment:
This needs to be rendered even if showLabel is false. We need to add/remove a 'u-sr-only' class if showLabel is true.